pkgs/os-specific/linux/tcp-wrappers/default.nix

```diff
@@ -20,6 +20,7 @@ in stdenv.mkDerivation rec {
   prePatch = ''
     tar -xaf $debian
     patches="$(cat debian/patches/series | sed 's,^,debian/patches/,') $patches"
+    substituteInPlace Makefile --replace "AR	= ar" "AR = ${stdenv.cc.targetPrefix}ar"
```

Comment:
FWIW it might be best (if you disagree nevermind it's basically the same) if instead of patching the Makefile we just add `"AR:=${stdenv.cc.targetPrefix}ar"`,  or `"AR:=$(AR)"` actually, to makeFlags?

Reply:
I have absolutely no opinion. I will happily do whatever you all deem best.

Reply:
Wow, I am slightly embarrassed to admit that I had no idea that `make` variables given on the command-line override definitions in the Makefile. Yes, overriding seems like a significantly better option.

Reply:
Or is this `:=` overriding `=`?

Reply:
To be clear, I didn't know this either, and I approve of @dtzWill's approach.

Reply:
`"AR:=$(AR)"` is especially nifty.